x-pack/filebeat/module/panw/panos/ingest/pipeline.yml

```diff
@@ -6,6 +6,13 @@ processors:
      field: message
      target_field: log.original
 
+# Get the correct timezone from the IETF header
+ - grok:
+     field: _temp_.ietf_header
+     patterns:
+       - '%{YEAR}-%{MONTHNUM}-%{MONTHDAY}[T ]%{HOUR}:?%{MINUTE}(?::?%{SECOND})?%{ISO8601_TIMEZONE:event.timezone}?'
+     on_failure: [{"append": {"field": "error.message", "value": "{{ _ingest.on_failure_message }}"}}]
```

Comment:
Instead of `on_failure` I think this needs `ignore_failure: true` because not all events will have an ISO8601 timestamp (like the rfc3164 syslog messages).

Reply:
I pushed an update for this.